opengrok-tools/src/main/python/opengrok_tools/deploy.py

```diff
@@ -27,77 +27,135 @@
 import tempfile
 from shutil import copyfile
 from zipfile import ZipFile
+import xml.etree.ElementTree as ET
 
 from .utils.log import get_console_logger, get_class_basename, \
     fatal
 from .utils.parsers import get_baseparser
 
+WEB_XML = 'WEB-INF/web.xml'
+DEFAULT_CONFIG_FILE = '/var/opengrok/etc/configuration.xml'
+
 """
  deploy war file
 
 """
 
 
-def repack_war(logger, sourceWar, targetWar, configFile, defaultConfigFile):
+class ProcessingException(Exception):
+    pass
+
+
+def insert_file(input_xml, insert_xml_file):
```

Comment:
how about moving this function to a separate file in utils?

Reply:
I think this is so specific to this script that it is not worth it.